kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -282,13 +291,32 @@ public static ConfigDef newConfigDef() {
           Type.INT,
           S3_PART_RETRIES_DEFAULT,
           Importance.MEDIUM,
-          "Number of upload retries of a single S3 part. Zero means no retries.",
+          "Maximum number of retry attempts for failed requests. Zero means no retries. "
+              + "An actual number of attempts is determined by the S3 client based on multiple "
+              + "factors, including, but not limited to - "
+              + "the value of this parameter, type of exception occurred, "
+              + "a number of attempts to retry failed request(s) etc.",
           group,
           ++orderInGroup,
           Width.LONG,
           "S3 Part Upload Retries"
       );
 
+      configDef.define(
+          S3_RETRY_BACKOFF_CONFIG,
+          Type.LONG,
+          S3_RETRY_BACKOFF_DEFAULT,
+          Importance.LOW,
+          "How long to wait in milliseconds before attempting the first retry "
+              + "of a failed S3 request. Upon a failure, this connector may wait up to twice as "
+              + "long as the previous wait, up to the maximum number of retries."
+              + "This avoids retrying in a tight loop under failure scenarios.",
```

Comment:
Need a space after the period on line 312.

Reply:
Thanks, fixed.